src/pages/current/Current.tsx

```diff
@@ -50,14 +50,20 @@ const Current = () => {
     setRoomInfo,
   ] = useState<RoomTypes>(initialRoomInfoData);
 
-  const { data } = useGetRoomInfo(roomUUID);
+  const { data, isError } = useGetRoomInfo(roomUUID);
 
   useEffect(() => {
     if (data) {
       setRoomInfo(data);
     }
   }, [data]);
 
+  useEffect(() => {
+    if (isError) {
+      navigate('/error');
+    }
+  }, [isError]);
+
```

Comment:
이 부분을 사용한다면 아래처럼 바꾸는 게 좋을 것 같아요!
그리고 나중에 말씀드리려고 했는데 방 생성 할 때에나 404 화면을 띄울 때 url은 변경하지 않되 안의 내용만 바꾸는 형식으로 진행하는 게 좋을 것 같아요 (지금처럼 `/roomStart`, `/roomCalendar`, `/roomTimer` 등등 분리하지 않고 `/createRoom` 과 같은 동일한 url 안에서 컴포넌트만 바뀌도록)

```suggestion
  useEffect(() => {
    if (isError) {
      navigate(`${ROUTES.ERROR}`);
    }
  }, [isError]);

```

Reply:
방생성 페이지 퍼널 형식으로 바꾸는건 어떨까요?

토스 슬래시 링크 참고할게요!
https://www.youtube.com/watch?v=NwLWX2RNVcw&t=379